contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
@theuni 
I am playing around right now with the `help2man` Makefile method and it seems it indeed only
runs `help2man`:  *if* the manpage files don't exist yet **OR** a string in the source cpp file changed (I have no idea how it does this - blackmagic?).

So if the man pages exist and the help options matches the source files, the end user can build  without having `help2man` installed. If this confirms it seems much better than the shell script... 

But it has some side effects, now the commit id is back in the manpage. And some other small problem which my script was working around (matching copyright footer for all manpages)